todo_app/trello.py

```diff
@@ -0,0 +1,155 @@
+import requests
+from dataclasses import dataclass
+
+
+TRELLO_PREFIX = 'trello_'
+
+
+def remove_trello_prefix_if_exists(id):
```

Comment:
I can't see why a trello item might have a `trello_` prefix - does this ever get used?